var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -47,6 +48,13 @@ def headers(self):
         headers.directories = [self.prefix.include.freetype2]
         return headers
 
+
+class CMakeBuilder(CMakeBuilder):
+    def cmake_args(self):
+        return [self.define("FT_WITH_ZLIB", True), self.define("FT_WITH_BZIP2", True)]
+
+
+class AutoToolsBuilder(AutotoolsBuilder):
```

Comment:
Not sure if it makes a difference: `Autotools` instead of `AutoTools`?

Reply:
This has been removed all together in the latest revisions.